### PR TITLE
Code optimisations for PHP8

### PR DESCRIPTION
**Pull request by GromNaN**
I'd like to propose this minor changes, mostly suggested by PHPStorm.

### CODE REVIEW

File: src/Psr6Store.php [hunk 1]
```diff
@@ -227,7 +227,7 @@ public function unlock(Request $request): bool
 
         try {
             $this->locks[$cacheKey]->release();
-        } catch (LockReleasingException $e) {
```
Comment (GromNaN) [inline src/Psr6Store.php:230]: [Catch by exception type only](https://php.watch/versions/8.0/catch-exception-type)

File: src/Psr6Store.php [hunk 2]
```diff
@@ -100,7 +100,7 @@ public function __construct(array $options = [])
         $this->options = $resolver->resolve($options);
         $this->cache = $this->options['cache'];
         $this->lockFactory = $this->options['lock_factory'];
-        $this->hashAlgorithm = version_compare(PHP_VERSION, '8.1.0', '>=') ? 'xxh128' : 'sha256';
+        $this->hashAlgorithm = \PHP_VERSION_ID >= 80100 ? 'xxh128' : 'sha256';
```
Comment (GromNaN) [inline src/Psr6Store.php:103]: The expression is evaluated by the compiler.

File: src/Psr6Store.php [hunk 3]
```diff
@@ -508,13 +508,13 @@ private function restoreResponse(array $cacheData): ?Response
 
         // BC
         if (\is_string($value)) {
-            $value = ['expires' => 0, 'contents' => $value];
```
Comment (GromNaN) [inline src/Psr6Store.php:511]: The key `expires` is not used.

**Comment by Toflar** [comment]
LGTM, wdyt @alexander-schranz?

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
Looks fine from my side the only case I have not enough knowledge about is the deletion of `expires: 0` part.

**Comment by Toflar** [comment]
> The only case I have not enough knowledge about is the deletion of `expires: 0` part.

That's indeed not needed :)

Thanks @GromNaN - released as 4.1.1: https://github.com/Toflar/psr6-symfony-http-cache-store/releases/tag/4.1.1

**Pull request merged**